src/batched/KokkosBatched_Util.hpp

```diff
@@ -825,8 +825,8 @@ KOKKOS_INLINE_FUNCTION auto subview_wrapper(
 template <class ViewValueType, class ViewType>
 KOKKOS_INLINE_FUNCTION ViewValueType
 access_view_bounds_check(ViewType v, int m, int n, const BoundsCheck::Yes &) {
-  return v(KOKKOSKERNELS_MACRO_MIN(m, v.extent_int(0)),
-           KOKKOSKERNELS_MACRO_MIN(n, v.extent_int(1)));
+  if (m < v.extent_int(0) && n < v.extent_int(1)) return v(m, n);
+  return (ViewValueType)0.0F;
```

Comment:
@e10harvey thanks for PR! Can this be changed to an if-else statement? On some compilers (I can't recall specifically, likely some version(s) of clang) the second return statement might trigger an unreachable code warning (and error out with -Werror)

Reply:
Thanks for the review. We're looking at the performance difference between using the macros versus using and if-else. If the if-else provides better performance, we'll go with that.